### ISSUE TITLE
#1576 ShowWindow(hWnd, SW_HIDE); Null pointer problem

### ISSUE THREAD

**Issue by Obsidian200** [MicrosoftEdge/WebView2Feedback#2391]
Excuse me. I am a newbie.
I have the same problem #1576 
`ShowWindow(hWnd, SW_HIDE);
...
CreateCoreWebView2EnvironmentWithOptions(pcwStrRuntimeFolder, nullptr, nullptr, Callback<ICoreWebView2CreateCoreWebView2EnvironmentCompletedHandler>([hWnd, lpszUrl](HRESULT result, ICoreWebView2Environment* env) -> HRESULT {
	env->CreateCoreWebView2Controller(hWnd, Callback<ICoreWebView2CreateCoreWebView2ControllerCompletedHandler>(
	// ^^^^ env is NULL!!
		...
	).Get());
    ...
).Get());`

![image](https://user-images.githubusercontent.com/31143076/165234584-c57fd3dc-77a8-4d46-9d1e-e2917086555c.png)

I have tried the following methods. The problem still exists.What should I do please?
Thanks

> Thanks for your reply, but we have just solved this problem.
> 
> We did not call "CoInitializeEx()" before CreateCoreWebView2EnvironmentWithOptions(). When we called "CoInitializeEx()", we got an available env parameter!
> 
> Now we can take the next step.

**Comment by Obsidian200** [comment]
Because the page loads with a white screen. So I am trying to create the hidden window. Show the window in ICoreWebView2NavigationCompletedEventHandler. But I ran into this problem: env = NULL.
I used another solution. First create the display window. Hide the window in ICoreWebView2CreateCoreWebView2ControllerCompletedHandler. Show the window again in ICoreWebView2NavigationCompletedEventHandler. This solution is working. But I would like to ask what is the cause of the first problem? Thanks!
